examples/opencv_rc.py

```diff
@@ -0,0 +1,541 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from threading import Event
+from enum import IntEnum, auto
+import numpy as np
+import cv2 as cv
+import pycozmo as pc
+from pynput import keyboard as kbd
+
+
+# Declare an enumeration providing all the directions in which Cozmo can move
+class Direction(IntEnum):
+    NONE = auto()
+    FORWARD = auto()
+    BACKWARD = auto()
+    LEFT = auto()
+    RIGHT = auto()
+
+
+# Declare a flag telling the program's main loop to stop
+GO_ON = True
```

Comment:
Why not declaring this as member of OpencvRC?

Reply:
To be honest, there is no reasons for that to be here. Some oversight on my end. Since I have to rework the class structure anyway, I will be sure to put it where it needs be.